shared/resourcemapping/src/test/java/com/google/cloud/opentelemetry/resource/ResourceTranslatorTest.java

```diff
@@ -88,6 +88,60 @@ private static Stream<Arguments> provideOTelResourceAttributesToMonitoredResourc
             Map.ofEntries(
                 new SimpleEntry<>(
                     ResourceAttributes.CLOUD_PROVIDER, ResourceAttributes.CloudProviderValues.GCP),
+                new SimpleEntry<>(
```

Comment:
Just confirming here: The newly added tests here are test cases that contain GCP Platform values, correct ?

Reply:
It is correct.